### PR TITLE
done fixes in settings and changed privacy url

### PR DESCRIPTION
**Pull request by khuzama98**

### CODE REVIEW

**Review by osancus — CHANGES_REQUESTED** [review]
I believe you don't have prettier installed on your VSCode or you are not using VSCode. The code is reformated which makes unnecessary changes to code. Can you please fix that, see .prettierrc.js file.

File: covidWallet/app/screens/SettingsScreen.js [hunk 1]
```diff
@@ -1,91 +1,114 @@
...
+import React, {useState} from 'react';
+import {StyleSheet, Text, View, FlatList, Linking} from 'react-native';
+import {TextTypeView, BooleanTypeView} from '../components/ShowTypesView';
+import HeadingComponent from '../components/HeadingComponent';
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons';
 
 var settingLocalData = {
   GENERAL: {
     Agent: {
       value: 'Iphone',
```
Comment (osancus) [inline covidWallet/app/screens/SettingsScreen.js:10]: Change this to 'Phone'

File: covidWallet/app/screens/SettingsScreen.js [hunk 2]
```diff
@@ -1,91 +1,114 @@
...
       type: 'Text',
-      key: '11'
+      key: '11',
     },
     Network: {
       value: 'Soverin Staging Network',
```
Comment (osancus) [inline covidWallet/app/screens/SettingsScreen.js:15]: Change this to 'BCovrin Test Network'

**Review by osancus — CHANGES_REQUESTED** [review]
This is how the image looks on iOS. Can you please make sure that the back button is transparent and the setting is not so top screen aligned but instead at a significant distance from the list below. Thank you!

IMG_197BF9D20411-1

**Comment by khuzama98** [comment]
@osancus I have done the changes again please review it and let me know, Thanks.

**Review by osancus — APPROVED** [review]

**Pull request merged**